### PR TITLE
Nonprofit Updates

### PR DESCRIPTION
**Pull request by jordanafung**

### CODE REVIEW

File: source/about/subscription.rst [hunk 1]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
 Who's eligible?
 ~~~~~~~~~~~~~~~~
 
-To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.
+Organizations applying for a non-profit license must meet **ALL** of the following requirements:
+- Be able to provide documentation that establishes a recognized nonprofit status.
```
Comment (cwarnermm) [inline source/about/subscription.rst:26]:
```suggestion
- Be able to provide documentation that establishes the organization as a recognized nonprofit status.
```

File: source/about/subscription.rst [hunk 2]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
...
+- Have no affiliation with government, academic, commercial, or religious entities.
+- Be unable to afford a commercial Mattermost license.
+- Be willing to let us use their logo for promotional purposes.
+- Be willing to pay a $250 processing fee for a three-year, Self-hosted Professional license contract for up to 1000 users.
```
Comment (cwarnermm) [inline source/about/subscription.rst:30]:
```suggestion
- Be willing to pay a $250 processing fee for a three-year, self-hosted Professional license contract for up to 1000 users.
```

File: source/about/subscription.rst [hunk 3]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
...
+
+If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>` instead.
+
+If you represent an open source community or project that is not hosted under a recognized nonprofit but would like a commercial license for Mattermost, please email community@mattermost.com with your organization’s needs and requirements.
```
Comment (cwarnermm) [inline source/about/subscription.rst:34]:
```suggestion
If you represent an open source community or project that is not hosted under a recognized nonprofit, but would like a commercial license for Mattermost, please email community@mattermost.com with your organization’s needs and requirements.
```

File: source/about/subscription.rst [hunk 4]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
...
 
 How to apply?
 ~~~~~~~~~~~~~
 
-To apply for the Mattermost Nonprofit License, `please complete this form <https://docs.google.com/forms/d/e/1FAIpQLSe2OUQGOQAni74FBXgAOooYsUTGbLn_VXFYEKUthg9Wmmiebg/viewform>`__.
+To apply for the Mattermost Nonprofit License, `please complete this form <https://mattermost.typeform.com/to/gbM6xGkS>`.
+Please note that it may take up to 6-8 weeks for us to respond to your request.
```
Comment (cwarnermm) [inline source/about/subscription.rst:40]:
```suggestion
Please note that it may take up to eight weeks for us to respond to your request.
```

**Review by cwarnermm — COMMENTED** [review]
Great update, @jordanafung! I've provided editorial feedback inline for you to consider.

**Comment by jordanafung** [comment]
> Great update, @jordanafung! I've provided editorial feedback inline for you to consider.

Thanks Carrie - suggestions accepted!

File: source/about/subscription.rst [hunk 5]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
...
+
+If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>` instead.
+
+If you represent an open source community or project that is not hosted under a recognized nonprofit, but would like a commercial license for Mattermost, please email community@mattermost.com with your organization’s needs and requirements.
 
 How to apply?
 ~~~~~~~~~~~~~
 
-To apply for the Mattermost Nonprofit License, `please complete this form <https://docs.google.com/forms/d/e/1FAIpQLSe2OUQGOQAni74FBXgAOooYsUTGbLn_VXFYEKUthg9Wmmiebg/viewform>`__.
+To apply for the Mattermost Nonprofit License, `please complete this form <https://mattermost.typeform.com/to/gbM6xGkS>`.
```
Comment (jordanafung) [inline source/about/subscription.rst:39]:
```suggestion
To apply for the Mattermost Nonprofit License, `please complete this form <https://mattermost.typeform.com/to/gbM6xGkS>`__.
```

File: source/about/subscription.rst [hunk 6]
```diff
@@ -22,12 +22,22 @@ Following the three-year free subscription, the institution can renew the licens
...
 
-To be eligible, an organization needs to be an official nonprofit or charity; as well as non-government, non-academic, and non-commercial in nature, with no religious affiliation; and that would otherwise be unable to afford the commercial version of Mattermost software. If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.
+Organizations applying for a non-profit license must meet **ALL** of the following requirements:
+- Be able to provide documentation that establishes the organization as a recognized nonprofit status.
+- Have no affiliation with government, academic, commercial, or religious entities.
+- Be unable to afford a commercial Mattermost license.
+- Be willing to let us use their logo for promotional purposes.
+- Be willing to pay a $250 processing fee for a three-year, self-hosted Professional license contract for up to 1000 users.
+
+If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>` instead.
```
Comment (jordanafung) [inline source/about/subscription.rst:32]:
```suggestion
If your organization doesn’t fit this description, we suggest that you purchase a `commercial license <https://mattermost.com/pricing/>`__ instead.
```

File: source/about/subscription.rst [hunk 7]
```diff
@@ -10,24 +10,35 @@ Mattermost Cloud is a software-as-a-service product. You can sign up for free, f
 Mattermost Nonprofit License program
 --------------------------------------
 
-The Mattermost Nonprofit License enables eligible charities, open source projects, and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering towards advancing their missions with special nonprofit pricing.
+The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted `:doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```
Comment (cwarnermm) [inline source/about/subscription.rst:13]:
```suggestion
The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```

File: source/about/subscription.rst [hunk 8]
```diff
@@ -10,24 +10,35 @@ Mattermost Cloud is a software-as-a-service product. You can sign up for free, f
 Mattermost Nonprofit License program
 --------------------------------------
 
-The Mattermost Nonprofit License enables eligible charities, open source projects, and nonprofit organizations to apply the benefits of the self-hosted Mattermost Professional offering towards advancing their missions with special nonprofit pricing.
+The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering </about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```
Comment (cwarnermm) [inline source/about/subscription.rst:13]:
```suggestion
The Mattermost Nonprofit License enables nonprofit organizations who are unable to afford our commercial licenses to apply the benefits of the self-hosted :doc:`Mattermost Professional offering <about/editions-and-offerings.html#mattermost-professional>` towards advancing their missions with special nonprofit pricing.
```

**Review by cwarnermm — APPROVED** [review]
The one link that isn't working is something we're investigating further.

**Pull request merged**